docs/sphinx/source/whatsnew/v0.9.2.rst

```diff
@@ -18,6 +18,8 @@ Bug fixes
   where passing localized timezones with large UTC offsets could return
   rise/set/transit times for the wrong day in recent versions of ``ephem``
   (:issue:`1449`, :pull:`1448`)
+* Added Townsend-Powers monthly snow loss model: :py:func:`pvlib.snow.loss_townsend`
+  (:issue:`1246`, :pull:`1251`, :pull:`1468`)
```

Comment:
Let's move this from `Bug fixes` to `Enhancements` :)

Reply:
Ah, right, thank you.